src/clarity-angular/datagrid/datagrid.html

```diff
@@ -36,7 +36,7 @@
 
     <!-- Custom placeholder overrides the default empty one -->
     <ng-content select="clr-dg-placeholder"></ng-content>
-    <clr-dg-placeholder *ngIf="!placeholder"></clr-dg-placeholder>
+    <clr-dg-placeholder *ngIf="!placeholder && items.displayed.length == 0"></clr-dg-placeholder>
```

Comment:
We still have to support datagrids without a fixed height, though. They should expand as much as needed without needing to scroll. That's where the placeholder still had to do some work. Admittedly it didn't work well with multi-line rows, and we had that on our backlog.

A solution now would be to force the datagrid height on initial load with the first page, and keep it fixed from there on. That's a bit debatable, because a datagrid that didn't scroll on page 1 might start scrolling on page 2, even though the consumer never wanted it to scroll.